taskt/Core/Automation/Commands/CreateDataTableCommand.cs

```diff
@@ -55,6 +55,10 @@ public override void RunCommand(object sender)
         {
             var engine = (Core.Automation.Engine.AutomationEngineInstance)sender;
             var dataTableName = v_DataTableName.ConvertToUserVariable(sender);
+            if (dataTableName == "")
```

Comment:
What does this check do in this case?

Reply:
Apologies, I placed these comments terribly. Will make more visible next time.

Reply:
The way it was originally, the command assumed you were initializing the DataTable name within the command and then using that name and the new DataTable value to create a new engine variable object. However if you create the name variable in a separate command and then use it as the v_DataTableName, it looks for it using the ConvertToUserVariable method, finds it, and returns an empty string as its value. In this case, you wouldn't want the variable object's value, you'd want its name.

Reply:
That makes sense!